docs/current_docs/integrations/snippets/depot-hello.yml

```
@@ -0,0 +1,12 @@
+name: dagger
+on:
+  push:
+    branches: [main]
+
+jobs:
+  hello:
+    name: hello
+    runs-on: depot-ubuntu-22.04,dagger={{ version }}
```

```suggestion
    runs-on: depot-ubuntu-24.04,dagger={{ version }}
```
Either one is fine with me. The latest Ubuntu version is faster (regardless of Depot).